README.md

```diff
@@ -55,7 +56,8 @@ $ npm install @sap-ai-sdk/foundation-models
 
 This package incorporates generative AI orchestration capabilities into your AI activities in SAP AI Core and SAP AI Launchpad.
 
-To install the  Gen AI Hub package in your project, run:
+To install the Gen AI Hub package in your project, run:
+
 ```
 $ npm install @sap-ai-sdk/orchestration
```

Comment:
[pp] I would mention orchestration first as this is supposed to be the most prominent of the packages.